index.html

```diff
@@ -1,39 +1,121 @@
-				<h1>PROJECT STRUCTURE</h1>
-			</header>
-		</section>
+              <!-- Toggler/collapsibe Button -->
+              <button
+                class="navbar-toggler"
+                type="button"
+                data-toggle="collapse"
+                data-target="#collapsibleNavbar"
+              >
+                <span class="navbar-toggler-icon"></span>
+              </button>
 
-		<section class="skills">
-		</section>
+              <!-- Navbar links -->
+              <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav ml-auto">
+                  <li class="nav-item active">
+                    <a class="nav-link" href="#">Home</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Services</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Work</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="#">Contact</a>
+                  </li>
+                </ul>
+              </div>
+            </nav>
+            <div class="row hero">
+              <div class="col-lg-7">
+                <div class="hero-content">
+                  <div class="content-wrap">
+                    <p>Hello.</p>
+                    <h1><span>I'm</span> Alaba Badiru</h1>
+                    <div class="skills">
+                      <div class="skill">Designer</div>
+                      <div class="seperator"></div>
+                      <div class="skill">Developer</div>
+                      <div class="seperator"></div>
+                      <div class="skill">Artist</div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="col-lg-5">
+                <div class="hero-image"></div>
+              </div>
+            </div>
+            <div class="social-handle">
+              <div class="pint"><i class="fab fa-pinterest"></i></div>
+              <div class="behance"><i class="fab fa-behance"></i></div>
+              <div class="dribble"><i class="fab fa-dribbble"></i></div>
+            </div>
+          </div>
+        </header>
+      </section>
 
-		<section class="experience">
-		</section>
+      <section class="skills"></section>
 
-		<section class="education">
-		</section>
+      <section class="experience"></section>
 
-		<section class="testimonials">
-		</section>
+      <section class="education"></section>
 
-		<section class="contact-me">
-		</section>
-	</main>
+      <section class="testimonials"></section>
 
-	<footer></footer>
-</body>
+      <section class="contact-me"></section>
+    </main>
 
-</html>
\ No newline at end of file
+    <footer></footer>
+
+    <script
```

Comment:
Please put all your script tags in the header which will be fetched asynchronously. To understand _why_ this is the best approach read this [stack overflow question](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup)

Reply:
Ok got it. On it.